### PR TITLE
Updating gfsv15.3.3 with EUM bufr changes for OPS implementation

### PR DESCRIPTION
**Pull request by ilianagenkova**
Updated sorc/checkout.sh to pick gfsda.v15.3.3 ( EUM bufr changes )

### CODE REVIEW

File: sorc/checkout.sh [hunk 1]
```diff
@@ -21,7 +21,8 @@ if [[ ! -d gsi.fd ]] ; then
     rm -f ${topdir}/checkout-gsi.log
     git clone --recursive gerrit:ProdGSI gsi.fd >> ${topdir}/checkout-gsi.log 2>&1
     cd gsi.fd
-    git checkout gfsda.v15.3.2
+    #git checkout gfsda.v15.3.2
```
Comment (KateFriedman-NOAA) [inline sorc/checkout.sh]: Iliana, to keep things clean please remove the commented out checkout line (line 24) and remove the text comment on the next line. We only need to update the version # for the tag. Commit those adjustments to your fork branch and they'll appear in here. Reply to this comment when done and I'll submit the PR , thanks!

**Review by KateFriedman-NOAA — COMMENTED** [review]
Make requested adjustments prior to me submitting this. Thanks!

File: sorc/checkout.sh (hunk 1 above)
Comment (ilianagenkova) [reply sorc/checkout.sh]: Cleaned up the comments and pushed. Thanks!

Comment (KateFriedman-NOAA) [reply sorc/checkout.sh]: Perfect thanks!

**Pull request merged**